lib/WebpackConfig.js

```diff
@@ -222,6 +237,18 @@ class WebpackConfig {
         this.useReact = true;
     }
 
+    enableTypeScriptLoader(options = {}) {
+        this.useTypeScriptLoader = true;
+
+        for (const optionKey of Object.keys(options)) {
+            if (!(optionKey in this.typeScriptOptions)) {
+                throw new Error(`Invalid option "${optionKey}" passed to enableTypeScriptLoader(). Valid keys are ${Object.keys(this.typeScriptOptions).join(', ')}`);
+            }
```

Comment:
More about what we talked already. Is a good point and I will update the PR to take this insto account.